Cesium.Core/Exceptions/WipException.cs

```diff
@@ -0,0 +1,17 @@
+namespace Cesium.Core.Exceptions;
+
+public class WipException : CesiumException
+{
+    /// <summary>A marker value for an issue number not yet assigned.</summary>
+    public const int ToDo = -1;
+
+    public WipException(int issueNo, string additionalMessage)
+        : base($"This work is in progress. {additionalMessage}. See https://github.com/ForNeVeR/Cesium/issues/{issueNo}.")
+    {
+    }
+
+    public WipException(int issueNo)
+        : base($"This work is in progress. See https://github.com/ForNeVeR/Cesium/issues/{issueNo}.")
+    {
+    }
```

Comment:
```suggestion
    }
    
    public static WipException Unknown(string message) => new WipException(ToDo, message);
```

Reply:
Nope. I prefer people to explicitly mention `WipException.ToDo` in their code, for easier monitoring on the review stage.